{"name": "test_unsuccessful_otp_send_to_invalid_email", "status": "passed", "steps": [{"name": "This test verifies that OTP does not send to an invalid email", "status": "passed", "start": 1648797960740, "stop": 1648797960942}], "start": 1648797960740, "stop": 1648797960943, "uuid": "7f662669-4422-4c97-b4e0-7d88082d7439", "historyId": "965ea806611f8025d243ae1fac785daf", "testCaseId": "b691a4a0ed760957e87d93347b60160c", "fullName": "tests.api_testcases.verify_email.verify_email_test.TestVerifyEmail#test_unsuccessful_otp_send_to_invalid_email", "labels": [{"name": "parentSuite", "value": "tests.api_testcases.verify_email"}, {"name": "suite", "value": "verify_email_test"}, {"name": "subSuite", "value": "TestVerifyEmail"}, {"name": "host", "value": "pop-os"}, {"name": "thread", "value": "105835-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.api_testcases.verify_email.verify_email_test"}]}